src/pages/task-page/index.tsx

```diff
@@ -7,6 +7,7 @@ import { ToDoItem } from "../../interfaces/interfaces";
 
 function TaskPage() {
   const list = useSelector((state: RootState) => state.list.list);
+  localStorage.setItem("List", JSON.stringify(list))
```

Comment:
Nop, you only want to set the list whenever you add an item to the list, not here